firedrake/slate/slac/compiler.py

```diff
@@ -59,13 +59,12 @@
 BLASLAPACK_INCLUDE = None
 if not complex_mode:
     if COMM_WORLD.rank == 0:
-        petsc_variables = get_petsc_variables()
-        EIGEN_INCLUDE_DIR = petsc_variables.get("EIGEN_INCLUDE")
+        EIGEN_INCLUDE_DIR = get_eigen_include_dir()
```

Comment:
This is needed because Eigen doesn't have to be linked to PETSc during install so this variable doesn't actually show up in `petscvariables`. In the Spack install I put this variable in `config["libraries"]["EIGEN_INCLUDE_DIR"]`.